app/views/controllers/herbaria/search/new.erb

```diff
@@ -3,8 +3,10 @@ add_new_title(:search_object, :HERBARIA)
 container_class(:wide)
 %>
 
-<%= render(Components::SearchForm.new(
-      @search,
-      search_controller: controller,
-      local: true
-    )) %>
+<div id="herbaria_search_container">
```

Comment:
If you're adding this wrapper for the id, i think the search form component already generates a unique id that tells what type of search form it is.